Automattic/notifications-panel
# Update engines to node 10.4 and update dev dependencies using npm audit fix

This PR updates our expected major version of Node to help with ease of development, and also updates a few stale npm developer dependencies using `npm audit fix`.

### Testing Instructions

Test Development Container:
- `rm -rf node_modules`
- switch to Node v10.4
- `npm install`
- `npm start`
- Visit http://notifications.localhost:8888/
- Verify that there are no regressions.

Calypso:
- navigate to your wp-calypso checkout
- checkout `update/notifications`
- `npm start`

or use the live link in 
https://github.com/Automattic/wp-calypso/pull/25466

- Verify that there are no regressions.


Standalone (iframe):
- sandbox a site and widgets (dm if you need more specific instructions)
- run `~/public_html/bin/notes/build-notifications.sh -c2.2.0-alpha.1`
- visit a blog site, and open the masterbar notifications there (not Calypso)
- verify that there are no regressions

see also PCYsg-elI-p2
